File: 002/002.php
<?php

$fibo = array(0, 1);
for ($i = 2; $i < 34; $i++)
    array_push($fibo, $fibo[$i-2] + $fibo[$i-1]);

echo(array_sum(array_filter(
    $fibo, function($x) { return $x % 2 == 0; })) . "\n");

File: 002/002.go
package main

import "fmt"

func main() {
	var fibo [36]int
	fibo[0], fibo[1] = 0, 1
	for i := 2; i < 34; i++ {
		fibo[i] = fibo[i-2] + fibo[i-1]
	}
	s := 0
	for _, x := range fibo {
		if x%2 == 0 {
			s += x
		}
	}
	fmt.Println(s)
}
